yarn-project/circuit-types/src/logs/l1_payload/encrypted_log_payload.ts

```diff
@@ -1,47 +1,36 @@
 import {
   AztecAddress,
-  Fq,
   Fr,
-  GrumpkinScalar,
-  type KeyValidationRequest,
+  type GrumpkinScalar,
   NotOnCurveError,
   PRIVATE_LOG_SIZE_IN_FIELDS,
   Point,
   PrivateLog,
   type PublicKey,
-  computeOvskApp,
   derivePublicKeyFromSecretKey,
 } from '@aztec/circuits.js';
 import { randomBytes } from '@aztec/foundation/crypto';
 import { BufferReader, type Tuple, numToUInt16BE, serializeToBuffer } from '@aztec/foundation/serialize';
 
 import { decrypt, encrypt } from './encryption_util.js';
-import { derivePoseidonAESSecret } from './shared_secret_derivation.js';
 
 // Below constants should match the values defined in aztec-nr/aztec/src/encrypted_logs/payload.nr.
 
-// Both the incoming and the outgoing header are 48 bytes../shared_secret_derivation.js
+const ENCRYPTED_PAYLOAD_SIZE_IN_BYTES = (PRIVATE_LOG_SIZE_IN_FIELDS - 1) * 31;
+
+// The incoming header is 48 bytes../shared_secret_derivation.js
```

Comment:
```suggestion
// The incoming header is 48 bytes
```

I don't really get why the file was linked there.